src/tmp/SampleEnginePlugin.ts

```diff
@@ -230,6 +232,45 @@ export class SampleEngine2 extends EngineApi.Engine {
 	}
 }
 
+export class SampleEngine3 extends EngineApi.Engine {
```

Comment:
New engine added, guaranteed to never throw violations. Useful for testing cases where no violations are thrown.